src/go/k8s/tests/e2e/admin-api-tls-client-auth/01-call-adminapi.yaml

```diff
@@ -23,7 +23,7 @@ spec:
             - -c
             - -ex
           args:
-            - curl https://cluster-tls-0.cluster-tls.$POD_NAMESPACE.svc.cluster.local:9644/metrics -v --cacert /etc/tls/certs/admin/ca.crt  --cert /etc/tls/certs/admin/tls.crt --key /etc/tls/certs/admin/tls.key
+            - curl https://cluster-tls-0.cluster-tls.$POD_NAMESPACE.svc.cluster.local:9645/metrics -v --cacert /etc/tls/certs/admin/ca.crt  --cert /etc/tls/certs/admin/tls.crt --key /etc/tls/certs/admin/tls.key
```

Comment:
This test verifies internal communication (inside k8s cluster). Why the port is `9645`, but not defined in the specification `9644`?

Reply:
Only the external port has TLS enabled.

Reply:
I modified the code to avoid this requirement.